db/app.model.js

```diff
@@ -46,9 +46,7 @@ exports.selectAllArticles = (query) =>{
 }
 
 exports.selectCommentsByArticleID = (article_id) =>{
-  if(typeof article_id === 'number'){
-    return Promise.reject({ status: 400, msg: "bad request" })
-  }
+ 
 const queryString = `SELECT comment_id,votes,created_at,author,body,article_id FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`
  return db.query(queryString,[article_id]).then(({rows}) => {
   if(rows.length === 0){
```

Comment:
If you have no comments on an article, it would be best to return an empty array rather than throwing an error. The request has been successful, there is just no comments to send back